docs/user/tutorials/01-10-setup-custom-domain-for-workload.md

```diff
@@ -7,32 +7,55 @@ This tutorial shows how to set up a custom domain and prepare a certificate requ
 
 ## Prerequisites
 
-* Deploy [a sample HTTPBin Service](./01-00-create-workload.md).
-* If you use a cluster not managed by Gardener, install the [External DNS Management](https://github.com/gardener/external-dns-management#quick-start) and [Certificate Management](https://github.com/gardener/cert-management) components manually in a dedicated namespace.
+* [Deploy a sample HTTPBin Service](./01-00-create-workload.md).
+* If you use a cluster not managed by Gardener, install the [External DNS Management](https://github.com/gardener/external-dns-management#quick-start) and [Certificate Management](https://github.com/gardener/cert-management) components manually in a dedicated namespace. SAP BTP, Kyma runtime clusters are managed by Gardener so you are not required to install any additional components.
 
 ## Steps
 
```

Comment:
Numbering doesn't look good in the steps section. It could be less confusing if the actual steps were formatted as headings 3, WDYT.
<img width="431" alt="Screenshot 2024-04-02 at 13 17 36" src="https://github.com/kyma-project/api-gateway/assets/107463569/1646a866-b8cc-444a-a62e-14d221af240c">

Reply:
I'm not sure. I agree that it doesn't look good, especially on GitHub, but can I use headings when the order of steps is important, and all of them are mandatory? 
It looks a bit better when rendered, though:
<img width="543" alt="Screenshot 2024-04-05 at 07 02 29" src="https://github.com/kyma-project/api-gateway/assets/80401180/8713e537-07ed-4aa5-8959-3145d18639bb">

Reply:
The tabs are not displayed correctly when I use a nested ordered list in them, so I've changed my mind and added headings instead :)